### ISSUE TITLE
Add Fortnightly Option To Recurring Events | Enhancement

### ISSUE THREAD

**Issue by KrychuTM** [RetroMusicPlayer/Paisa#302]
Would be awesome to have a "Fortnightly" option between "Weekly" and "Monthly" for periodic events in the Recurring section.

**Comment by BalooRJ1** [comment]
Agreed, a Biweekly would be very helpful with this app since many people get paid this way.

**Comment by targodan** [comment]
Can this maybe be generalized? I saw another app having a number field `n` with an option to "repeat every `n` <interval>" with "<interval>" being the selected interval (weeks, months, ...).

I e.g. could also use a half-yearly repetition. And I imagine every 3 month might also be useful.
